### DISCUSSION TITLE
Mini event : Le très ancien site d'artillerie de la vallée des rois

### DISCUSSION THREAD

**Discussion by romain22222** [Crownicles/Crownicles#2496] [Nouvelles idées]
Pour faire simple, une grille nous est présenté avec une case choisie au préalable par le jeu. On a 3 choix : 
- Ne pas la toucher (rien se passe)
- Observer la case (en gros aller dessus) -> Gain d'un item dont la rareté correspond au numéro libéré
- Marquer la case comme étant une mine -> Gain d'argent par rapport au nombre de numéros disponibles autour (plus y'en a plus ca rapporte.

Si on rate l'observation (y'a une mine là où on marche) alors on prend de lourds dégats (~90% de sa vie)
Si on marque une case sans mine, on prend une pénalité de temps (les personnes sur place vous interrogent sur votre potentiel lien avec certains ennemis du royaume)

La grille présentée est globale à tous les joueurs, une seule personne peut tomber sur la grille à la fois. La grille reset toutes les semaines.

Si une personne fait le dernier bon choix de la grille, elle gagne une grosse récompense (item leg+, etc.)
```
Lors de votre intrépide journée dans le désert, vous décidez de faire un détour vers le champ archéologique en cours. Une personne sur place vous propose de l'aider à sécuriser le site en vous demandant d'aller vérifier cette case, la pointant du doigt, en vous montrant les belles récompenses qui vous attendent.

Champ en cours : 

<affichage d'une grille de 6x6 en emotes>

<marcher>
<marquer>
<partir>
```

**Comment by BastLast** [comment]
Argh il essai de transformer draftbot en un jeu 2d

**Reply by romain22222** [reply]
Ménaaaaann, c est juste un plan du site, le joueur lui se déplace dans une zone 3d

**Reply by romain22222** [reply]
Et techniquement draftbot a déjà de la 2d avec la carte 🫠

**Reply by BastLast** [reply]
Forgot about that 🫠
